lib/helpers/coverage.js

```diff
@@ -35,6 +35,11 @@ module.exports.init = ({
 };
 
 module.exports.setCoverage = ({ path, method, status }) => {
+  if (!schema) {
```

Comment:
This used to break individual tests unless they initted coverage explicitly as well